config/locales/en_custom_errors.yml

```diff
@@ -0,0 +1,5 @@
+en:
+  doorkeeper:
+    errors:
+      messages:
+        my_message: 'This is my authorization custom message'
```

Comment:
> What about adding a new file in the same folder `en_custom_errors.yml`

I'd say this is not needed. Can we mock the `I18n.t` call in tests, so we don't need to add this far-away file for that one particular spec?